gradle/dependencies.gradle

```diff
@@ -18,7 +18,7 @@ ext {
   println("Navigation Native version: " + mapboxNavigatorVersion)
 
   version = [
-      mapboxMapSdk              : '10.11.0-rc.1',
+      mapboxMapSdk              : '10.11.0',
```

Comment:
Any changelog needed?

Reply:
a link to the maps release will be in Dependencies section